mzLib/Omics/Digestion/LysisProduct.cs

```diff
@@ -0,0 +1,192 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Omics.Modifications;
+
+namespace Omics.Digestion
+{
+    public class LysisProduct
```

Comment:
maybe digestion product rather than lysis product. lysis is confusing with cell lysis even though I know you are relating lysis to lytic.

Reply:
Changed